variables.tf

```diff
@@ -102,7 +102,7 @@ variable "esxi_devices" {
   type = map(object({
     name           = string # Short form hostname of system (vcf-ems-deployment-parameter.xlsx > Hosts and Networks Sheet > I6:L6)
     mgmt_ip        = string # Management Network IP address for VMK0 (vcf-ems-deployment-parameter.xlsx > Hosts and Networks Sheet > I7:L7)
-    reservation_id = string # Hardware reservation IDs to use for the VCF nodes. Each item can be a reservation UUID or `next-available`.
+    reservation_id = optional(string, "") # Hardware reservation IDs to use for the VCF nodes. Each item can be a reservation UUID or `next-available`.
```

Comment:
Looks good, but CI is complaining because of formatting.  Could you run `terraform fmt` on your branch and commit & push the changes?

Reply:
Done! :-)